pkg/controller/v1beta1/inferenceservice/rawkube_controller_test.go

```diff
@@ -1946,7 +1946,7 @@ var _ = Describe("v1beta1 inference service controller", func() {
 				},
 				URL: &apis.URL{
 					Scheme: "http",
-					Host:   fmt.Sprintf("%s-default.example.com", serviceName),
+					Host:   fmt.Sprintf("%s-predictor.%s.svc.cluster.local", serviceName, serviceKey.Namespace),
```

Comment:
Can we add a comment if this needs to be different for ODH? Or is this just something we haven't synced yet?